src/components/organisms/OcResource/OcResource.spec.js

```diff
@@ -88,4 +93,17 @@ describe("OcResource", () => {
     expect(wrapper.find(".parent-folder").find("a").exists()).toBeFalsy()
     expect(wrapper.find(".parent-folder").attributes("style")).toEqual("cursor: default;")
   })
+
+  it("displays parent folder name default if given", () => {
+    const wrapper = mount(Resource, {
+      propsData: {
+        resource: fileResourceWithParentFoldername,
+        isPathDisplayed: true,
+        parentFolderNameDefault: "Example parent folder name",
+      },
+      stubs,
+    })
+
+    expect(wrapper).toMatchSnapshot()
```

Comment:
Snapshot test is not necessary here. Can check the parent folder name text directly (compare with the default that was given in `propsData`)